pennylane/_grad.py

```diff
@@ -332,3 +334,130 @@ def _jacobian_function(*args, **kwargs):
         return jac[0] if unpack else jac
 
     return _jacobian_function
+
+
+# pylint: disable=too-many-arguments
+def vjp(f, params, cotangents, method=None, h=None, argnum=None):
+    """A :func:`~.qjit` compatible Vector-Jacobian product for PennyLane.
```

Comment:
To emphasis the support: 
```suggestion
    """A :func:`~.qjit` compatible Vector-Jacobian product of PennyLane programs.
```